doc/HACKING/EndOfLifeTor.md

```diff
@@ -0,0 +1,52 @@
+
+End of Life on an old release series
+------------------------------------
+
+Here are the steps that the maintainer should take when an old Tor release
+series reaches End of Life:
+
+=== 0. Preliminaries
+
+0. A few months before End of Life:
+   Write a deprecation announcement.
+   Send the announcement out with every new release announcement.
+
+1. A month before End of Life:
+   Send the announcement to tor-announce, tor-talk, tor-relays, and the
+   packagers.
+
+2. A few days before End of Life:
+   Get at least two of weasel/arma/Sebastian to
+   remove the old version number from their approved versions list.
+   Give them a few days to do this if you can.
```

Comment:
We could define a bit more in details this procedure because #27983 made it clear that the process is unclear. Historically, Roger would push a patch to the dirauth-conf git repository but lately it has been Nick asking weasel on IRC to do so and it would be pushed.

So I propose we continue towards that path with: "The stable maintainer notifies weasel/arma/Sebastian about the change considering it an official request". Then one of them pushes the update to the git repository and we move on.